static-site/components/nav/styles.css

```diff
@@ -0,0 +1,30 @@
+nav {
+  align-items: center;
+  display: flex;
+  height: 50px;
+  justify-content: space-between;
+  left: 0px;
+  margin: auto;
+  overflow: hidden;
+
+  a {
+    color: var(--darkGrey);
+    font-size: 16px;
```

Comment:
Suggestions:

1. Set font-size on `nav` so it's applicable to non-links as well
2. Use `var(--niceFontSize)`